rust/Cargo.toml

```diff
@@ -14,15 +14,16 @@ reqwest = { version = "0", features = [
 	"rustls-tls",
 ], default-features = false }
 # symphonia = { version = "0.5.2", features = ["all"] }
-symphonia = { git = "https://github.com/erikas-taroza/Symphonia", branch = "mp4-improvements", features = [
-	"all",
-] }
+symphonia = { git = "https://github.com/erikas-taroza/Symphonia", branch = "mp4-improvements", features = ["all"] }
```

Comment:
Use the [`mp4-opus-improvements`](https://github.com/erikas-taroza/Symphonia/tree/mp4-opus-improvements) branch for Symphonia dependencies.